waspc/examples/todoApp/sample.env

```diff
@@ -0,0 +1,2 @@
+TEST_ENV_VAR="I am test"
```

Comment:
What is this for, this env file?

Reply:
I added the existing `.env` file to our `.gitignore` so we can now add a `DATABASE_URL`. So I just made the old one a `sample.env` people can copy/paste to bootstrap theirs. In the todoApp, we do check for and print `TEST_ENV_VAR` so I figured it warranted a sample file. Thoughts?